homeassistant/__init__.py

```diff
@@ -27,7 +27,7 @@
 DOMAIN = "homeassistant"
 
 # How often time_changed event should fire
-TIMER_INTERVAL = 10  # seconds
+TIMER_INTERVAL = 3  # seconds
```

Comment:
This one is basically hinged on the shitty Wink API.  I can't find a way to get pushed, and I want to trigger lights on door entry.  Chicken?  Egg?

If we could get push from wink it would be easy, but waiting 10s to go to the bathroom at 2am sucks.